### ISSUE TITLE
[Task]: Set inputs when hitting back button through history state

### ISSUE THREAD

**Issue by rylew1** [HHS/simpler-grants-gov#1479]
### Summary

In #1458 we wrote to and read from the query param string - so that input selections were kept in sync with the URL.

When setting inputs, it will update the url query param string with `history.pushState` - when you hit back through each query param update, it will just update the URL/query param string through what you're already selected (your history).

When you hit the back button it should also updates the inputs, not just the URL.

### Acceptance criteria

- When hitting back button through query param / input state - the input selections are updated

Blocked by #1303 and usability sessions with users to see what the behavior is before updating

**Comment by rylew1** [comment]
Proposing to defer this as it seems lower priority @andycochran  @acouch

**Comment by acouch** [comment]
This was implemented in [#2067](https://github.com/HHS/simpler-grants-gov/issues/2067) . The search page URL now changes every time you change the input. The update maintains the history so the browser back and forward button work properly through the URL changes.
